docs/tutorials/mutelist.md

```diff
@@ -108,23 +114,23 @@ If you want to mute failed findings only in specific regions, create a file with
             Resources:
               - "*"
 
-## Default AWS Mutelist
+### Default Mutelist
```

Comment:
I would not remove AWS, since the default mutelist is only available for AWS

Reply:
The same again, I've included an AWS section to enclose this.